### ISSUE TITLE
MermaidTailStore.com now accepting Bitcoin Cash

### ISSUE THREAD

**Issue by vizanto** [acceptbitcoincash/acceptbitcoincash#271]
Requesting to add MermaidTailStore.com to the Retail category. 

Details follow:

```yml

- name: Mermaid Tail Store
  url: https://MermaidTailStore.com
  twitter: tail_store
  facebook: mermaidtailstore
  img: https://mermaidtailstore.com/logo_256.jpg
  bcc: yes
  btc: no
  othercrypto: no
```

**Comment by t0p0lin0** [comment]
Added and merged in with pull request #273 -- thank you for contributing!

**Comment by vizanto** [comment]
Thanks! Website hasn't updated yet though, any idea why?
